### PR TITLE
Rename IS_UNITS as SI_UNITS

### PR DESCRIPTION
**Pull request by joseivanlopez**

### CODE REVIEW

**Comment by jreidinger** [comment]
sorry, but this is wrong. It is really SI units see https://en.wikipedia.org/wiki/International_System_of_Units

File: src/lib/y2storage/disk_size.rb [hunk 1]
```diff
@@ -38,7 +38,7 @@ class DiskSize
 
     UNITS = ["B", "KiB", "MiB", "GiB", "TiB", "PiB", "EiB", "ZiB", "YiB"]
     # International System units
```
Comment (imobachgs) [inline src/lib/y2storage/disk_size.rb]: np: I would fix this comment to read like `International System of Units (SI)` so the renaming is more clear.

**Comment by jreidinger** [comment]
my fault i see it other way :)

File: src/lib/y2storage/disk_size.rb (hunk 1 above)
Comment (joseivanlopez) [inline src/lib/y2storage/disk_size.rb]: Ok, thanks.

**Pull request merged**